R/formulas.R

```diff
@@ -195,55 +173,82 @@ prb <- function(assessed, sale_price, na.rm = FALSE) {
 
 
 
-##### ki_mki #####
+##### MKI_KI #####
+
+# Calculate the Gini cofficients needed for KI and MKI
+calc_gini <- function(assessed, sale_price) {
```

Comment:
Here I've moved the Gini code into its own function so it can be shared between `ki()` and `mki()`.